dotnet/src/SemanticKernel.Abstractions/Contents/ImageContent.cs

```diff
@@ -17,9 +17,14 @@ public sealed class ImageContent : KernelContent
     public Uri? Uri { get; set; }
 
     /// <summary>
-    /// The image binary data.
+    /// The image media type.
     /// </summary>
-    public BinaryData? Data { get; set; }
+    public string? MediaType { get; set; }
+
+    /// <summary>
+    /// The image data.
+    /// </summary>
+    public ReadOnlyMemory<byte>? Data { get; set; }
```

Comment:
@dersia, just a heads-up that we are changing the type of the Data property that you've added.

Reply:
thanks for letting me know. interestingly enough I did implement it this way in my first implementation and it was suggested to switch to BinaryData during discussion. with all the dependencies that Abstraction had it didn't seem like a bad idea so I changed the implementation.

but good that we get rid of dependencies and I am happy that this stays in the way it was designed originally.